templates/components/products/price-range.html

```diff
@@ -1,21 +1,28 @@
-{{#and price.retail_price_range.min.without_tax price.retail_price_range.max.without_tax}}
-    <div class="price-section price-section--withTax" {{#if schema_org}}itemprop="offers" itemscope itemtype="http://schema.org/Offer"{{/if}}>
-        <span data-product-price-without-tax class="price price--rrp">{{price.retail_price_range.min.without_tax.formatted}} - {{price.retail_price_range.max.without_tax.formatted}}</span>
-        {{#if schema_org}}
-            <meta itemprop="availability" content="{{product.availability}}">
-            <meta itemprop="itemCondition" itemtype="http://schema.org/OfferItemCondition" content="http://schema.org/{{product.condition}}Condition">
-            <div itemprop="priceSpecification" itemscope itemtype="http://schema.org/PriceSpecification">
-                <meta itemprop="minPrice" content="{{price.retail_price_range.min.without_tax.value}}"  />
-                <meta itemprop="price" content="{{price.retail_price_range.min.without_tax.value}}">
-                <meta itemprop="maxPrice" content="{{price.retail_price_range.max.without_tax.value}}"  />
-                <meta itemprop="priceCurrency" content="{{currency_selector.active_currency_code}}">
-                <meta itemprop="valueAddedTaxIncluded" content="true">
+{{#and price.retail_price_range.min.with_tax price.retail_price_range.max.with_tax}}
+    <div class="price-section price-section--withTax rrp-price--withTax" {{#if schema_org}}itemprop="offers" itemscope itemtype="http://schema.org/Offer"{{/if}}>
+        {{theme_settings.pdp-retail-price-label}}
+        <span data-product-rrp-price-with-tax class="price price--rrp">{{price.retail_price_range.min.with_tax.formatted}} - {{price.retail_price_range.max.with_tax.formatted}}</span>
+    </div>
+    {{else}}
+        {{#if price.with_tax}}
+            <div class="price-section price-section--withTax rrp-price--withTax" style="display: none;">
+                {{theme_settings.pdp-retail-price-label}}
+                <span data-product-rrp-with-tax class="price price--rrp">
+                    {{price.rrp_with_tax.formatted}}
+                </span>
             </div>
         {{/if}}
-    </div>
 {{/and}}
 {{#and price.price_range.min.with_tax price.price_range.max.with_tax}}
+    <div class="price-section price-section--withTax non-sale-price--withTax" style="display: none;">
```

Comment:
Interesting that you chose to default with `display: none` here, should this also have an `unless` block around it?

Reply:
At least for right now a sale price should _never_ display if a price range is being displayed. At least that is my thinking. I want the element to be on the page but want to avoid having a `sale_price` from the lowest priced variant display next to a price range.